docs/install/2-install-server-with-docker.rst

```diff
@@ -131,6 +138,13 @@ Here is a minimalistic docker-compose file content that can be used to deploy th
                 volumes:
                     - ./resources/com.inmanta.license:/etc/inmanta/license/com.inmanta.license
                     - ./resources/com.inmanta.jwe:/etc/inmanta/license/com.inmanta.jwe
```

Comment:
Unfortunately I cannot use env variables for that in the example. I guess this will come later.